src/components/elementView/elementTab.tsx

```diff
@@ -295,6 +296,22 @@ class ElementTab extends React.Component<ElementTabProps, ElementViewState> {
       console.log(this.state.isOpen);
     });
   };
+
+  private createChangePlan = (
+    changePlan: ChangePlan,
+    headers: any
+  ): Promise<any> => {
+    return axios
+      .post(API_ROOT + "api/change-plans/add", changePlan, headers)
+      .then(res => {
+        console.log("success");
+
+        this.handleDataUpdate(true);
+        this.handleClose();
+        this.addSuccessToast("Successfully created ChangePlan!");
```

Comment:
no space in ChangePlan typo ?

Reply:
fixeddd to dynamically get the success message  from endpoint